nucliadb_models/nucliadb_models/metadata.py

```diff
@@ -249,9 +249,23 @@ class TokenSplit(BaseModel):
 
 
 class ParagraphAnnotation(BaseModel):
-    classifications: List[Classification] = Field(Classification, min_items=1)
+    classifications: List[Classification] = []
     key: str
 
+    @root_validator()
+    def _validate_classifications(cls, values):
+        classifications = values.get("classifications") or []
+        if len(classifications) < 1:
```

Comment:
`if len(classifications) == 0` looks more understandable if you want to check emptiness here